.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*.{cs, py}]
```

Comment:
maybe we should remove the other file types so it doesn't interfere with other projects?

Reply:
You mean just switching it to something like this?
```
[*]
indent_size = 4
```

Reply:
i was thinking more python only.